roles/nova-data/tasks/ssh.yml

```diff
@@ -38,3 +38,17 @@
 
 - name: verify ssh among compute nodes
   command: sudo -u nova -H /bin/sh -c {{ nova.state_path }}/bin/verify-ssh
+
+- name: Install nova ssh keys for root
+  authorized_key:
+    user: root
+    key: "{{ hostvars[item].nova_user.ssh_public_key }}"
+  with_items: groups['compute']
+
+- name: root known_hosts
+  template:
+    src: root/ssh/known_hosts
+    dest: "/root/.ssh/known_hosts"
+    owner: root
+    group: root
+    mode: 0600
```

Comment:
libvirt logs in as root on the other compute nodes, so we've gotta make sure we have functional ssh keys, known_hosts setup for root.

I wanted to reuse the keys that were created rather than make new ones.  This might not be what you want, so I'm commenting and pointing it out.  

We're in a known-working state - so if need be, can make new keys.